emotion_ferplus/README.md

```diff
@@ -17,17 +20,16 @@ The model is trained in CNTK, using the cross entropy training mode. You can fin
 
 ## Model input and output
 ### Input
-The model expects a grayscale input image of the shape `(1x64x64)`
+shape `(1x1x64x64)`
```

Comment:
did the model change from before or just a documentation fix?

Reply:
documentation fix. I have updated the sample input data to be 4d to include the batch dimension